src/pages/global/Home/index.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react'
 import workImage from 'assets/images/work.jpg'
+import { makeStyles } from '@material-ui/core/styles';
+//components
+import { CategoryContainer } from "../../../components";
```

Comment:
can be used as
```
import { CategoryContainer } from "components";
```

I've added a setting to make src the root path.

Reply:
Ok, will make changes.